xml/security-barbican.xml

```diff
@@ -271,19 +246,101 @@ ansible-playbook -i hosts/localhost ready-deployment.yml</screen>
     </para>
    </step>
   </procedure>
- </section>
- <section xml:id="sec.barbican.kmip">
-  <title>KMIP Plug-in Support</title>
+  <bridgehead renderas="sect4">Upgrade Master Key Configuration</bridgehead>
```

Comment:
Please avoid adding more bridgeheads to these documents -- use real sections instead. If there is a nesting level issue, bridgeheads don't really avoid it, they just hide it. We will need to reorganize large parts of these documents at some point anyway, because of their completely impractical nesting level.
Also, for titles, use gerund (`ing`).